### DISCUSSION TITLE
Mirai 留言板

### DISCUSSION THREAD

**Discussion by Him188** [mamoe/mirai#696] [General]
感谢你对 Mirai 的支持。

你可以在这里留下对 Mirai 的看法和未来的发展建议，请畅所欲言。

**Comment by takayama-lily** [comment]
好的

**Comment by Karlatemp** [comment]
嘤

**Comment by MikuNyanya** [comment]
不知道说啥
还是

\兔子万岁/

吧

感谢你们的努力

**Comment by keainye** [comment]
https://github.com/mamoe/mirai/blob/dev/docs/guide_quick_start.md 此页面下, "添加依赖"板块下, "直接导入Jar包"下, 有两个下载链接. 其中, 第二个链接的文字中有一个单词拼错了, "mirai-qqandriod" 应该为 "mirai-qqandroid"

**Comment by zhufree** [comment]
感谢

**Comment by anaivebird** [comment]
听说禁言匿名在测试了吗，进展如何

**Comment by Him188** [comment]
> 听说禁言匿名在测试了吗，进展如何

完成了，再等时机成熟就合并

**Comment by anaivebird** [comment]
请问禁言匿名的代码commit了吗

**Reply by Him188** [reply]
现在对 `AnonymousMmber.mute()` 即可

**Comment by Nambers** [comment]
用winapi或者其他什么东西在win上监测非exit命令退出(如异常退出或关闭cmd窗口)，然后阻塞并执行onDisable()方法并等待结束再退出？
或者做个后台线程，如果前台线程退出就代替主线程执行onDisable再退出？

**Reply by Nambers** [reply]
现在好像就exit命令退出才有onDisable方法

**Comment by igo9go** [comment]
未来会有计划搞微信协议吗

**Reply by Him188** [reply]
目前没有

**Comment by Fred2371218232** [comment]
求求出点Java插件的教程吧，哪怕是一个“Hello World”的 Example孩子最起码也知道Java的import有哪些和格式是什么样了，太难了o(╥﹏╥)o

**Comment by litouzi** [comment]
好炫

**Comment by xiaoyv404** [comment]
为什么 MiraiConsoleGradlePlugin 内的 configureSourceSet 方法下的 languageSettings 要 useExperimentalAnnotation 在前面，而不是 optIn
Gradle 在新版本已经不提倡使用 useExperimentalAnnotation 了， useExperimentalAnnotation  放在前面导致每次编译的时候都会有个警告。。。
![like this](https://user-images.githubusercontent.com/63290381/164838789-52ae0b21-c50c-4c47-b24a-2ca45edc5469.png)

**Reply by Him188** [reply]
之前是为了保证使用旧版本 Kotlin 时也能正常编译（mirai-console-gradle 是在 Kotlin 写 optIn 之前写的）

**Reply by xiaoyv404** [reply]
> 之前是为了保证使用旧版本 Kotlin 时也能正常编译（mirai-console-gradle 是在 Kotlin 写 optIn 之前写的）

那为何不吧 optln 放在前面捏，优先新版本
如果找不到才使用 useExperimentalAnnotation 
还是说，有其他问题

**Reply by Him188** [reply]
2.11 会这么做

**Reply by xiaoyv404** [reply]
> 2.11 会这么做

好的，十分感谢！看这个警告看了好久了
